src/test/java/com/keyboardbuilder/enterprise/EnterpriseApplicationTests.java

```diff
@@ -1,13 +1,34 @@
 package com.keyboardbuilder.enterprise;
 
+import com.keyboardbuilder.enterprise.dto.KeySwitches;
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
 @SpringBootTest
 class EnterpriseApplicationTests {
 
     @Test
     void contextLoads() {
     }
 
+
+    @Test
+    void verifyKeySwitchesProperties()  {
+        String type = "Write the type";
+        int weight = 22;
+        String brand ="Write the brand name";
+
+        KeySwitches keySwitches = new KeySwitches();
+        keySwitches.setType(type);
+        assertEquals(type, keySwitches.getType());
+
+        keySwitches.setWeight(weight);
+        assertEquals(weight, keySwitches.getWeight());
+
+        keySwitches.setBrand(brand);
+        assertEquals(brand, keySwitches.getBrand());
+    }
+
```

Comment:
There was no test created for the project. So, I created a test and it passes.
Source: Lecture Videos.